### ISSUE TITLE
Upgrade leaves ceibo dependency in Bower

### ISSUE THREAD

**Issue by nbibler** [san650/ember-cli-page-object#190]
Upgrading from 1.1.0 left the ceibo dependency in the bower.json file. Perhaps this should be removed when upgrading the library now that it's provided via npm.

Related to #188.

**Comment by san650** [comment]
Continuing the conversation from #188 

> I can... I just worry if it will errantly remove a dependency that someone else might've added to Bower. Hard to say if "ceibo" was added for page object or something else after the fact. :-/

It's true, maybe we can print an after install message or something instead of removing the dependency, although I don't believe any project is using Ceibo as a bower dependency.

**Comment by san650** [comment]
ember-cli-test-loader throws a deprecation notice on build, we could do the same.

```
DEPRECATION: ember-cli-test-loader should now be included as an NPM module with version 1.1.0 or greater.
```

**Comment by san650** [comment]
This issue doesn't make sense anymore.
